src/app.py

```diff
@@ -144,77 +154,111 @@ def query(
     return QueryResponseSchema(query_hash=job.hash)
 
 
-@app.get("/api/matches/{hash}", response_model=MatchesSchema)
+@app.get("/api/matches/{hash}", response_model=MatchesSchema, tags=["stable"])
 def matches(
     hash: str, offset: int = Query(...), limit: int = Query(...)
 ) -> MatchesSchema:
+    """
+    Gets a lost of matched files, along with metadata tags and other
```

Comment:
I don't like the `Gets` word in here (and in following comments) as it doesn't imply if it inputs or outputs said data, but I guess it's fine.

Reply:
Changed to `returns` everywhere